stories/custom_components/search.js

```diff
@@ -12,4 +12,16 @@ const Search = ({ searchPlaceholder, onChange }) => {
   );
 };
 
+export const SearchWithValue = ({ searchPlaceholder, onChange, value }) => {
+  return (
+    <TextField
+      value={value}
+      placeholder={searchPlaceholder}
+      onChange={onChange}
+      fullWidth
+      style={{ margin: "17px 0 0 0" }}
```

Comment:
use const for this value and use it in both

Reply:
Fixed